QuizBowlDiscordScoreTracker/Web/wwwroot/js/main.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
+var nameDisplay = document.getElementById("name");
+var sound1 = document.getElementById("sound1");
+
+var shouldEmitSound = true;
+
+function removeBarrier() {
+    document.getElementById("interactBarrier").style.display = "none";
+}
+
+connection.on("PlayerBuzz", function (name) {
+    document.body.style.backgroundColor = "red";
+    nameDisplay.textContent = name;
+
+    if (shouldEmitSound) {
+        sound1.play();
```

Comment:
Just to make sure: did you create the buzz sound? If not, do we have the rights to use it?

Reply:
I grabbed it from soundjay.com with the following terms of use


>  You are allowed to use the sounds free of charge and royalty free in your projects (such as films, videos, games, presentations, animations, stage plays, radio plays, audio books, apps) be it for commercial or non-commercial purposes.
> 
> But you are NOT allowed to
> - post the sounds (as sound effects or ringtones) on any website for others to download, copy or use
> - use them as a raw material to create sound effects or ringtones that you will sell, distribute or offer for downloading
> - sell or license the sounds (as sound effects or ringtones) to anyone else
> - use the sounds in songs or productions such as 'soothing sounds', 'relaxing music', 'natural sounds', 'sounds for sleep', 'meditation video', 'relaxing video', 'stress relief', etc.
> - link directly to individual sound files on our server
> - distribute the sounds in apps or computer programs that are clearly sound related in nature (such as sound machine, funny sounds app, sound therapy app, ringtone maker, etc.). If you are developing such programs, contact us for licensing options.
> - distribute the sounds in apps or computer programs that use the sounds as the program's sound resource library for other people's use (such as animation creator, video editing app, song maker software, ebook creator, etc.). If you are developing such programs, contact us for licensing options.
> - use the sounds to create so-called 'video sound effects'. An example would be a video titled 'dog sound effect' that simply plays a dog sound effect or a video titled 'relaxing rain' or '2 hours of relaxing rain' that plays a rain sound effect.
> 
> If you use the sound effects, please consider giving us a credit but it's not required.